### DISCUSSION TITLE
Are the start and end parameters for observations functional?

### DISCUSSION THREAD

**Discussion by kpm-at-hfi** [weather-gov/api#337] [Q&A] [answered]
I tried using the `start` and `end` GET parameters to select a week or a day in the past and got only an empty

Example of failing past observation request/response (using the OpenAPI form at https://www.weather.gov/documentation/services-web-api).

Request:
```
curl -X GET "https://api.weather.gov/stations/KFCM/observations?start=2020-05-25T00%3A00%3A00Z&end=2020-05-26T00%3A00%3A00Z" -H "accept: application/ld+json"
```

Response:
```
{
  "title": "Unexpected Problem",
  "type": "https://api.weather.gov/problems/UnexpectedProblem",
  "status": 500,
  "detail": "An unexpected problem has occurred.",
  "instance": "https://api.weather.gov/requests/493c3a1d-f87e-407f-ae2c-24483f5aab63",
  "correlationId": "493c3a1d-f87e-407f-ae2c-24483f5aab63"
}
```

**Comment by scadergit** [comment, marked as answer]
Terribly sorry for the late response. We had some ingest issues with obervation data around that time, but it's too far back to verify. I was able to use your example url to get observations using a recent time window. Observation records are kept for 7 days.

Let me know if you need any other assistance.
